mmdet/models/anchor_heads/anchor_head.py

```diff
@@ -94,8 +95,17 @@ def forward_single(self, x):
         bbox_pred = self.conv_reg(x)
         return cls_score, bbox_pred
 
-    def forward(self, feats):
-        return multi_apply(self.forward_single, feats)
+    def forward(
+            self,
+            feats: List[torch.Tensor],
+    ) -> Tuple[List[torch.Tensor], List[torch.Tensor]]:
```

Comment:
We plan to add type hints for the whole project in the future. At this time, it is better to keep the original style.

Reply:
Ok.

This change is leaked from JIT support PR. Hints are required to customize type inference in TorchScript.